builders/build/templates/overview.md

```diff
@@ -1,58 +1,48 @@
 ---
 title: Templates
-description: Tanssi includes templates to kick-start the development of an appchain, one for a Substrate-oriented runtime and another featuring full EVM (Ethereum) support.
+description: Tanssi includes templates to kick-start the development of an appchain offering options such as a basic one and another featuring full EVM (Ethereum) support.
```

Comment:
```suggestion
description: Tanssi includes templates to kick-start the development of an appchain with full EVM (Ethereum) support and minimal engineering overhead.
```

Reply:
I think it was discussed that we just mention the EVM template btw (considering the previous PRs we merged)